Anlab.Core/Data/ApplicationDbContext.cs

```diff
@@ -79,6 +79,14 @@ protected override void OnModelCreating(ModelBuilder builder)
             // Customize the ASP.NET Identity model and override the defaults if needed.
             // For example, you can rename the ASP.NET Identity table names and more.
             // Add your customizations after calling base.OnModelCreating(builder);
+
+            // prevent duplicate FKs when running migrations
```

Comment:
@jpknoll couldn't figure out how to get this to work.  https://docs.microsoft.com/en-us/aspnet/core/migration/1x-to-2x/identity-2x suggests you can do this and then your user.roles relationship will still work, but it didn't for me, so I'm using the userManager service instead